# -*- coding: utf-8 -*-

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_giderTable(object):
    def setupUi(self, giderTable):
        giderTable.setObjectName("giderTable")
        giderTable.resize(814, 468)
        self.tbl_gider = QtWidgets.QTableWidget(giderTable)
        self.tbl_gider.setGeometry(QtCore.QRect(10, 0, 541, 401))
        self.tbl_gider.setObjectName("tbl_gider")
        self.tbl_gider.setColumnCount(4)
        self.tbl_gider.setHorizontalHeaderLabels(["Gider No", "Tarih", "Gider Türü", "Gider Tutarı"])
        self.tbl_gider.setEditTriggers(QtWidgets.QTableWidget.NoEditTriggers)
        #Table column resize
        header = self.tbl_gider.horizontalHeader()       
        header.setSectionResizeMode(0, QtWidgets.QHeaderView.ResizeToContents)
        header.setSectionResizeMode(1, QtWidgets.QHeaderView.ResizeToContents)
        header.setSectionResizeMode(2, QtWidgets.QHeaderView.Stretch)
        header.setSectionResizeMode(3, QtWidgets.QHeaderView.ResizeToContents)
        self.btn_anamenu = QtWidgets.QPushButton(giderTable)
        self.btn_anamenu.setGeometry(QtCore.QRect(410, 430, 75, 23))
        self.btn_anamenu.setObjectName("btn_anamenu")
        self.cmb_yil = QtWidgets.QComboBox(giderTable)
        self.cmb_yil.setGeometry(QtCore.QRect(690, 50, 91, 31))
        self.cmb_yil.setObjectName("cmb_yil")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.cmb_yil.addItem("")
        self.label_3 = QtWidgets.QLabel(giderTable)
        self.label_3.setGeometry(QtCore.QRect(690, 10, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_3.setFont(font)
        self.label_3.setAlignment(QtCore.Qt.AlignCenter)
        self.label_3.setObjectName("label_3")
        self.label_2 = QtWidgets.QLabel(giderTable)
        self.label_2.setGeometry(QtCore.QRect(570, 10, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.label_2.setObjectName("label_2")
        self.cmb_ay = QtWidgets.QComboBox(giderTable)
        self.cmb_ay.setGeometry(QtCore.QRect(580, 50, 91, 31))
        self.cmb_ay.setObjectName("cmb_ay")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.cmb_ay.addItem("")
        self.label_4 = QtWidgets.QLabel(giderTable)
        self.label_4.setGeometry(QtCore.QRect(620, 130, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_4.setFont(font)
        self.label_4.setAlignment(QtCore.Qt.AlignCenter)
        self.label_4.setObjectName("label_4")
        self.txt_Gider = QtWidgets.QLineEdit(giderTable)
        self.txt_Gider.setGeometry(QtCore.QRect(602, 170, 141, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.txt_Gider.setFont(font)
        self.txt_Gider.setAlignment(QtCore.Qt.AlignCenter)
        self.txt_Gider.setReadOnly(True)
        self.txt_Gider.setObjectName("txt_Gider")

        self.retranslateUi(giderTable)
        QtCore.QMetaObject.connectSlotsByName(giderTable)

    def retranslateUi(self, giderTable):
        _translate = QtCore.QCoreApplication.translate
        giderTable.setWindowTitle(_translate("giderTable", "Gider Tablosu"))
        self.btn_anamenu.setText(_translate("giderTable", "AnaMenü"))
        self.cmb_yil.setItemText(0, _translate("giderTable", "2022"))
        self.cmb_yil.setItemText(1, _translate("giderTable", "2023"))
        self.cmb_yil.setItemText(2, _translate("giderTable", "2024"))
        self.cmb_yil.setItemText(3, _translate("giderTable", "2025"))
        self.cmb_yil.setItemText(4, _translate("giderTable", "2026"))
        self.cmb_yil.setItemText(5, _translate("giderTable", "2027"))
        self.cmb_yil.setItemText(6, _translate("giderTable", "2028"))
        self.label_3.setText(_translate("giderTable", "Yıl Seçiniz"))
        self.label_2.setText(_translate("giderTable", "Ay Seçiniz"))
        self.cmb_ay.setItemText(0, _translate("giderTable", "1"))
        self.cmb_ay.setItemText(1, _translate("giderTable", "2"))
        self.cmb_ay.setItemText(2, _translate("giderTable", "3"))
        self.cmb_ay.setItemText(3, _translate("giderTable", "4"))
        self.cmb_ay.setItemText(4, _translate("giderTable", "5"))
        self.cmb_ay.setItemText(5, _translate("giderTable", "6"))
        self.cmb_ay.setItemText(6, _translate("giderTable", "7"))
        self.cmb_ay.setItemText(7, _translate("giderTable", "8"))
        self.cmb_ay.setItemText(8, _translate("giderTable", "9"))
        self.cmb_ay.setItemText(9, _translate("giderTable", "10"))
        self.cmb_ay.setItemText(10, _translate("giderTable", "11"))
        self.cmb_ay.setItemText(11, _translate("giderTable", "12"))
        self.label_4.setText(_translate("giderTable", "Gider"))
        self.txt_Gider.setText(_translate("giderTable", "0"))

